pkg/validator/validator_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validator
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+type fakeJiraEndpoint struct {
+	issue       []byte
+	matchResult []byte
+}
+
+func (srv *fakeJiraEndpoint) Server() *httptest.Server {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/issue/", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		fmt.Fprintf(w, "%s", srv.issue)
+	})
+	mux.HandleFunc("/jql/match", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		fmt.Fprintf(w, "%s", srv.matchResult)
+	})
+
+	return httptest.NewServer(mux)
+}
+
+func TestValidation(t *testing.T) {
+	t.Parallel()
+
+	match := Match{
```

Comment:
I don't see this used in the test?

Reply:
It is used to instantiate MatchResult. Was thinking having a variable could make it more readable, moved the value inline now.